### PR TITLE
[Symfony] Add constant int value support on WebTestCaseAssertIsSuccessfulRector

### PR DESCRIPTION
**Pull request by JohJohan**
Add failing test for #250 

I think the fix might be in ValueResolver from PhpParser as we use `isValue` to check if the constant is equal to int. But maybe we should also resolve the constant values?

### CODE REVIEW

**Comment by johanadivare** [comment]
I also added a test for WebTestCaseAssertResponseCodeRectorTest which uses same ValueResolver so also fails

**Comment by JohJohan** [comment]
@TomasVotruba maybe you can give a hit on where to look?

**Comment by johanadivare** [comment]
I did some more digging and found that `resolveExprValueForConst` in `ValueResolver` doesn't give the constant value back but you get `Symfony\Component\HttpFoundation\Response::HTTP_OK` maybe that is expected and i should use different functionality to get the value? Maybe @samsonasik have some insight on this?

**Comment by johanadivare** [comment]
> You need to register constant into stub class at https://github.com/rectorphp/rector-symfony/tree/main/stubs/Symfony/Component/HttpFoundation

I changed the [Response.php](https://github.com/rectorphp/rector-symfony/blob/main/stubs/Symfony/Component/HttpFoundation/Response.php) in my vendor folder but i still get value `Symfony\Component\HttpFoundation\Response::HTTP_OK"`

**Comment by JohJohan** [comment]
@TomasVotruba can you add hacktober label `HACKTOBERFEST-ACCEPTED`? just like in https://github.com/rectorphp/rector-symfony/pull/252#issuecomment-1265038904 :)

File: tests/Rector/MethodCall/WebTestCaseAssertIsSuccessfulRector/Fixture/using_constants.php.inc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class UsingConstant extends WebTestCase
+{
+    public function test()
+    {
+        $client = $this->client;
+        $response = $client->getResponse();
```
[the remark this answers is not in the document]
Comment (samsonasik) [reply tests/Rector/MethodCall/WebTestCaseAssertIsSuccessfulRector/Fixture/using_constants.php.inc:32]: No need, that should by Rector Dead Code SetList.

**Comment by samsonasik** [comment]
@JohJohan please update PR title, to something like 

```
[Symfony] Add constant int value support on WebTestCaseAssertIsSuccessfulRector
```

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
THank you :clap:

**Comment by JohJohan** [comment]
> @TomasVotruba can you add hacktober label `HACKTOBERFEST-ACCEPTED`? just like in [#252 (comment)](https://github.com/rectorphp/rector-symfony/pull/252#issuecomment-1265038904) :)

@TomasVotruba thanks for merging can you have a look at ^?

**Comment by TomasVotruba** [comment]
@JohJohan Sure, done :hugs:

**Pull request merged**